### PR TITLE
Followup edits to PR2646

### PR DESCRIPTION
**Pull request by bfallonf**
#2646

@adellape @tpoitras @ahardin-rh Should just need a peer review. Please?

### CODE REVIEW

**Comment by bfallonf** [comment]
[rev_history]
|xref:../web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Added xref:../web_console_customization.adoc#configuring-navigation-menus[Configuring Navigation Menus] section.
%

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -173,21 +173,33 @@ assetConfig:
...
+dropdown menus. You can add additional menu items to these using the
+link:https://github.com/openshift/angular-extension-registry[angular-extension-registry].
+
+The available extension points are:
+
+* nav-help-dropdown - the help icon dropdown menu, visible at desktop screen widths
+* nav-user-dropdown - the user dropdown menu, visible at desktop screen widths
+* nav-dropdown-mobile - the single menu for top navigation items at mobile screen widths
+
+The following example extends the `nav-help-dropdown` menu, with a name of '<myExtensionModule>':
```
Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: Maybe put nav-help-dropdown in bold and <myExtensionModule> in backticks?

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -173,21 +173,33 @@ assetConfig:
...
+
+* nav-help-dropdown - the help icon dropdown menu, visible at desktop screen widths
+* nav-user-dropdown - the user dropdown menu, visible at desktop screen widths
+* nav-dropdown-mobile - the single menu for top navigation items at mobile screen widths
+
+The following example extends the `nav-help-dropdown` menu, with a name of '<myExtensionModule>': 
+
+[NOTE]
+====
+'<myExtensionModule>' is a placeholder name. Each dropdown menu extension must
```
Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: `<myExtensionMondule>`

**Comment by ahardin-rh** [comment]
FYI: I updated the rev history to include /install_config in the path. :surfer:

**Comment by bfallonf** [comment]
Thanks @ahardin-rh I'll merge this. 

(and thanks for fixing the rev history :bee: )

**Pull request merged**